### PR TITLE
three: improved ShapeUtils definition

### PR DESCRIPTION
**Pull request by mrpink17**

### CODE REVIEW

File: types/three/three-core.d.ts [hunk 1]
```diff
@@ -6643,10 +6643,9 @@ export namespace SceneUtils {
 }
 
 export namespace ShapeUtils {
-    export function area(contour: number[]): number;
-    export function triangulate(contour: number[], indices: boolean): number[];
-    export function triangulateShape(contour: number[], holes: any[]): number[];
-    export function isClockWise(pts: number[]): boolean;
+    export function area(contour: Vector2[]): number;   
+    export function triangulateShape(contour: Vector2[], holes: any[]): number[];
+    export function isClockWise(pts: Vector2[]): boolean;
```
Comment (RyanCavanaugh) [inline types/three/three-core.d.ts]: The spelling is `isClockwise`

https://threejs.org/docs/#api/extras/ShapeUtils.isClockwise

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
See comment

**Comment by mrpink17** [comment]
Hi,
it seems THREE documentation is wrong.
Please look at the sources:
https://github.com/mrdoob/three.js/blob/master/src/extras/ShapeUtils.js line 26

isClockWise: function ( pts ) {

File: types/three/three-core.d.ts (hunk 1 above)
Comment (mrpink17) [reply types/three/three-core.d.ts]: Hi,
it seems THREE documentation is wrong.
Please look at the sources:
https://github.com/mrdoob/three.js/blob/master/src/extras/ShapeUtils.js line 26

isClockWise: function ( pts ) {

**Pull request closed without merging**